tests/v2/validation/provisioning/rke2/custom_cluster_test.go

```diff
@@ -163,6 +166,18 @@ func (c *CustomClusterProvisioningTestSuite) ProvisioningRKE2CustomCluster(exter
 					clusterToken, err := clusters.CheckServiceAccountTokenSecret(client, clusterName)
 					require.NoError(c.T(), err)
 					assert.NotEmpty(c.T(), clusterToken)
+
+					if tt.hardening.Hardened {
+						err = hardening.HardeningNodes(client, tt.hardening.Hardened, nodes, tt.nodeRoles)
+						require.NoError(c.T(), err)
+						hardenCluster := clusters.HardenK3SRKE2ClusterConfig(clusterName, namespace, "", "", kubeVersion, nil)
```

Comment:
Same last nit-pick I had in the forwardport. Can we make sure the code is more readable, just like how you have in the dynamic version? Everything else lgtm!